distribution/ltp/lite/configs/RHELKT1LITE.20180926

```diff
@@ -620,8 +648,8 @@ memcmp01 memcmp01
 memcpy01 memcpy01
 
 migrate_pages01 migrate_pages01
-# See https://github.com/linux-test-project/ltp/issues/407
-#migrate_pages02 migrate_pages02
+migrate_pages02 migrate_pages02
```

Comment:
It doesn't look like the issue mentioned above is fixed yet. IIRC the test wasn't failing 100% of the time. Do we know for sure the test is not failing anymore on those specific machines?

Reply:
Same as above